modules/ROOT/pages/cloudhub-api.adoc

```diff
@@ -70,13 +71,17 @@ When you call the REST APIs, the following status codes are returned:
 |===
```

Comment:
Suggest using a definition list rather than a table for all 2-col tables that comprise an item and its description or definition. GLOBAL.